### PR TITLE
rpcserver: ForwardingHistory end_time defaults to time.Now()

### PR DESCRIPTION
**Pull request by tyzbit**
Closes simple version of #3240.

#### Pull Request Checklist

- [X] If this is your first time contributing, we recommend you read the [Code
  Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
- [X] All changes are Go version 1.12 compliant
- [X] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
- [X] For new code: Code is accompanied by tests which exercise both
  the positive and negative (error paths) conditions (if applicable)
- [X] For bug fixes: Code is accompanied by new tests which trigger
  the bug being fixed to prevent regressions
- [X] Any new logging statements use an appropriate subsystem and
  logging level
- [X] Code has been formatted with `go fmt`
- [X] For code and documentation: lines are wrapped at 80 characters
  (the tab character should be counted as 8 characters, not 4, as some IDEs do
  per default)
- [X] Running `make check` does not fail any tests
- [X] Running `go vet` does not report any issues
- [X] Running `make lint` does not report any **new** issues that did not
  already exist
- [X] All commits build properly and pass tests. Only in exceptional
  cases it can be justifiable to violate this condition. In that case, the
  reason should be stated in the commit message.
- [X] Commits have a logical structure according to [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)

### CODE REVIEW

File: rpcserver.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/btcsuite/btcwallet/wallet/txauthor"
 	"github.com/coreos/bbolt"
 	"github.com/davecgh/go-spew/spew"
-	"github.com/grpc-ecosystem/go-grpc-middleware"
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```
Comment (halseth) [inline rpcserver.go]: unnecessary change?

File: rpcserver.go [hunk 2]
```diff
@@ -4602,12 +4602,18 @@ func (r *rpcServer) ForwardingHistory(ctx context.Context,
 
 	// If the start and end time were not set, then we'll just return the
 	// records over the past 24 hours.
-	if req.StartTime == 0 && req.EndTime == 0 {
+	if req.StartTime == 0 {
```
Comment (halseth) [inline rpcserver.go]: comment should be updated

File: rpcserver.go (hunk 1 above)
Comment (tyzbit) [reply rpcserver.go]: For some reason my editor (Atom) wanted that there.  I removed it.

File: rpcserver.go (hunk 2 above)
Comment (tyzbit) [reply rpcserver.go]: Updated comment.
